src/main/java/com/conveyal/gtfs/error/NewGTFSErrorType.java

```diff
@@ -11,6 +11,7 @@
     URL_FORMAT(Priority.MEDIUM, "URL format should be <scheme>://<authority><path>?<query>#<fragment>"),
     LANGUAGE_FORMAT(Priority.LOW, "Language should be specified with a valid BCP47 tag."),
     ILLEGAL_FIELD_VALUE(Priority.MEDIUM, "Fields may not contain tabs, carriage returns or new lines."),
+    // The error type below is an MTC-specific requirement.
```

Comment:
Let's organize these into sections instead of having an individual comment for potentially each error type.

Reply:
Did you have anything in mind for reorganizing?

Reply:
I'd say one section for errors that apply during all types of validation and another section for MTC-specific errors.

Reply:
How about this? https://github.com/conveyal/gtfs-lib/pull/276/files#diff-8946752ed48c4dd733a37388eec80ac8